src/status_im/i18n.cljs

```diff
@@ -52,80 +53,170 @@
     [status-im.translations.zh-yue :as zh-yue]
     [status-im.utils.js-resources :refer [default-contacts]]
     [taoensso.timbre :as log]
-    [clojure.string :as str]))
+    [clojure.string :as str]
+    [clojure.set :as set]))
 
 (set! (.-fallbacks rn-dependencies/i18n) true)
 (set! (.-defaultSeparator rn-dependencies/i18n) "/")
 
+;;;; translations
```

Comment:
looks like you searched and replaced coma and there was already 2 so now you have 4

Reply:
I marked it as heading comment according to [this](https://github.com/bbatsov/clojure-style-guide#four-semicolons-for-heading-comments), but not sure if you use those kind of comments, so just reverted it.